src/types/app.ts

```diff
@@ -152,6 +157,8 @@ export type AppProps = {
   currencyId: string
   currencies: { [currencyId: string]: Currency }
   navs: AppNavProps[]
+  options: Options
+  endedAt: string
```

Comment:
endedAt is timestamp with time zone or nullable in table
type should be Date | null

Reply:
Okay! Thanks